packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -60,6 +61,16 @@ const Input = ({
 
   const onKeyDown = ['number', 'decimal'].includes(type) ? numberOnKeyDown : undefined;
 
+  useEffect(() => {
+    if (openCalendar) {
+      try {
+        input.current.showPicker();
+      } catch (e) {
+        console.log(e);
+      }
+    }
+  }, []);
```

Comment:
@MwayaB 

Sorry if I wasn't clear in my last comment! The useEffect() hook just needs to have the dependent variables added to its dependencies so [openCalendar] would go where you had autoFocus previously in this line. The linting rule React made for it is called exhaustive-deps in case your interested... one of these days we should declare it to our linter.